app/helpers/sections_helper.rb

```diff
@@ -18,4 +21,9 @@ def header_path_for_section(section, phase, template)
                                             id: section.id)
     end
   end
+
+  def draggable_for_section(section, sections)
+    section == sections.first && section.template.latest? && section.template.draft?
```

Comment:
I don't think `.draft?` is enough here. 

These states really have more to do with the template family than individual templates. There are 3 separate states:
- **Unpublished**: There are no published versions for the family
- **Published**: The family has a published version (either the current or a previous version)
- **Draft**: The family has a published version, but it is NOT the current/latest version

The draft/unpublished distinction is largely for the UI. We use it to give a visual cue to the user that their changes are not yet available to users creating plans and, for funder templates, have not yet propagated to any customized copies.

I think you would want the section to be draggable when its both 'draft' and 'unpublished'. Could you not just check for modifiable? That might be more versatile and allow you to use this for any section.
`sections.template.latest? && section.modifiable?`

Reply:
Thanks for the suggestion and the background explanation.